src/components/editTestCase/groupCoverage/QdmGroupCoverage.tsx

```diff
@@ -55,11 +56,20 @@ const definitionFilterCondition = (statementResult, definitionCategory) => {
   );
 };
 
+const sdeFilter = (statementResult, definitionCategory) => {
```

Comment:
Looks like definitionCategory is never used

Reply:
@RohitKandimalla I used it before but forgot to remove it, good catch, thanks just removed it.